src/convertToObject.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const someMagic = sourceString.split(';')
+    .map(element => element.split(':'));
+
+  someMagic.forEach(string => {
+    if (string.length > 1) {
```

Comment:
nice, but one important semantic comment - `someMagic` is an array of arrays, right?
so forEach loop interacts not with the string, but with an array.
Let's keep code properly descriptive 😉

Reply:
True, fixed.